build/gradle.go

```diff
@@ -265,12 +265,13 @@ func (config *gradleRunConfig) GetCmd() *exec.Cmd {
 func handleGradleCommandProperties(tasks []string) []string {
 	var cmdArgs []string
 	for _, task := range tasks {
-		if !strings.HasPrefix(task, systemPropertiesFlag) && !strings.HasPrefix(task, projectPropertiesFlag) {
-			cmdArgs = append(cmdArgs, task)
-			continue
+		hasPropertiesFlag := strings.HasPrefix(task, systemPropertiesFlag) || strings.HasPrefix(task, projectPropertiesFlag)
+		isProperty := hasPropertiesFlag && strings.Contains(task, "=")
+		if isProperty {
```

Comment:
This logic can be exported to a new function with a descriptive name. This will improve the readability of the code & tests